src/clinical_etl/mohschemav3.py

```diff
@@ -340,7 +340,7 @@ def validate_treatments(self, map_json):
                         case "Surgery":
                             if "surgeries" not in map_json or len(map_json["surgeries"]) == 0:
                                 self.warn("Treatment type Surgery should have one or more surgery submitted")
-            elif prop == "treatment_start_date" and map_json["treatment_start_date"] is not None:
+            if prop == "treatment_start_date" and map_json["treatment_start_date"] is not None:
```

Comment:
Out of curiosity, is this change necessary? prop can't be treatment_type and treatment_start_date at the same time, so elif works too